training_methods/debug_training_method.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict
```

Comment:
What does this class do?

Reply:
Added a docstring.

Used for testing/debug purposes.